src/metadataID3v2.ml

```diff
@@ -165,23 +170,18 @@ let parse ?recode f : metadata =
           let n = try next_substring encoding data with Not_found -> 0 in
           let data = String.sub data n (String.length data - n) |> recode in
           tags := ("comment", data) :: !tags)
-        else if id.[0] = 'T' || id = "COMM" then (
+        else if id.[0] = 'T' then (
           let encoding = int_of_char data.[0] in
           let recode = recode encoding in
           let data = String.sub data 1 (len - 1) |> recode in
-          if id = "TLEN" then (
-            match int_of_string_opt data with
-              | Some n ->
-                  tags :=
-                    ("duration", string_of_float (float n /. 1000.)) :: !tags
-              | None -> ())
-          else tags := (normalize_id id, data) :: !tags)
```

Comment:
I removed this artificial conversion:
* The field could have any value
* We want to keep this library as un-opiniated as possible
* Float string representation are locale-dependent (for instance use of `.` vs `,`). The choice made by this code might be incompatible with the float format expected lated.